### PR TITLE
Issue #63: Added pydoc to ppb/scenes.py

### PR DESCRIPTION
**Pull request by Cnidarias**

### CODE REVIEW

File: ppb/scenes.py [hunk 1]
```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
...
+
+        :param event: unused
+        """
         _ = event
         self.running = False
         self.quit = True
 
     def __mouse_up__(self, event):
+        """
+        Does nothing
```
Comment (pathunstrom) [inline ppb/scenes.py]: "Do nothing"

File: ppb/scenes.py [hunk 2]
```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
...
         """
         Default case, override in subclass as necessary.
+
+        :return: Tuple specifying if the scene is running and what the scene class is
         """
         return self.running, {"scene_class": self.next}
 
     def __null__(self, event):
+        """
+        Does nothing
```
Comment (pathunstrom) [inline ppb/scenes.py]: Word as command. "Do nothing."

File: ppb/scenes.py [hunk 3]
```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
...
+
+        :param event: unused
+        """
         pass
 
     def __quit__(self, event):
+        """
+        Quits the current scene
```
Comment (pathunstrom) [inline ppb/scenes.py]: This doesn't actually quit the scene, it just sets it's running and quit values appropriately.

File: ppb/scenes.py [hunk 4]
```diff
@@ -9,6 +9,13 @@
 class BaseScene(Scene):
 
     def __init__(self, engine, **kwargs):
+        r"""
```
[the remark this answers is not in the document]
Comment (Cnidarias) [reply ppb/scenes.py]: The r makes it a raw string in order to preserve special characters that would be turned something else by the likes of Sphinx - however it should not be needed anymore as I changed the **kwargs comment style

File: ppb/scenes.py [hunk 5]
```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
         }
 
     def render(self):
+        """
+        Renders all the objects in the groups
```
Comment (pathunstrom) [inline ppb/scenes.py]: Should be worded as a command."Render all the objects"

File: ppb/scenes.py [hunk 6]
```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
...
+
+        :return: Iterable representing all the drawn rectangles
+        """
         window = self.engine.display
         rects = [g.draw(window, self.background) for g in self.groups.values()]
         return chain(*rects)
 
     def handle_event(self, event):
+        """
+        Calls the specified callback function from the callback map
```
Comment (pathunstrom) [inline ppb/scenes.py]: "Call the specified . . ."

File: ppb/scenes.py [hunk 7]
```diff
@@ -20,33 +27,65 @@ def __init__(self, engine, **kwargs):
...
+        """
+        Do nothing
+
+        :param event: unused
+        """
         pass
 
     def __quit__(self, event):
+        """
+        Sets running to False and quit to True
```
Comment (pathunstrom) [inline ppb/scenes.py]: "Set running"

**Comment by pathunstrom** [comment]
Closing for inactivity. I think other work has done this work as well. Please feel free to submit a new PR for other docstrings if interested.

**Pull request closed without merging**